container_test.go

```diff
@@ -294,9 +294,8 @@ func TestCycles(t *testing.T) {
 	require.NoError(t, c.Provide(c1))
 	require.NoError(t, c.Provide(c2))
 	err := c.Provide(c3)
-
 	require.Contains(t, err.Error(), "unable to Provide dig.Type3")
-	require.Contains(t, err.Error(), "dig.Type3 -> dig.Type1 -> dig.Type3")
+	require.Contains(t, err.Error(), "func(dig.Type1) dig.Type3 -> func(dig.Type2, dig.Type3) dig.Type1 -> func(dig.Type1) dig.Type3")
```

Comment:
The old message was only reading the graph with strictly single object node or constructor resolving in single node. but now we have graph with constructor that creates multiple return nodes. This message is more descriptive now since constructor can have multiple args and cycle can be formed on one of the return argument in the funcNode.